scripts/dkms.mkconf

```diff
@@ -30,6 +30,7 @@ PRE_BUILD="configure
   --with-spl=\${source_tree}/spl-\${PACKAGE_VERSION}
   --with-spl-obj=\${dkms_tree}/spl/\${PACKAGE_VERSION}/\${kernelver}/\${arch}
   --with-spl-timeout=300
+  --with-qat=\${ICP_ROOT}
```

Comment:
@daweiq is the QAT driver packaged using dkms?   If so it would make sense to first check for an installed dkms version and if detected provide that to `--with-qat` and `--with-qat-obj`.  Then additionally provide a way with an environment variable (or two) to specify a location.  Please do this in a way such that `-with-qat` isn't passed unconditionally to configure when it's not available.

Reply:
@behlendorf QAT driver only provides source code tar ball now. So, we cannot detect the QAT driver source code unless user specific it. After passing this environment variable to configure, the source file check inside configure file will validate the path and decide whether to include QAT or not.

Reply:
Fair enough, then please just update the PR so the `--with-qat` is only provided to `configure` when `ICP_ROOT` is defined.  You can use a modified versions of how `--enable-debug` is provided and check if the environment variable has a non-zero length.

Reply:
Done, patch updated.